### PR TITLE
Replaced vendored omnijson by the standard lib version

### PR DESCRIPTION
**Pull request by claudep**
Refs #273.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,16 +14,6 @@
...
     sys.exit()
 
-if sys.argv[-1] == 'speedups':
-    try:
-        __import__('pip')
-    except ImportError:
-        print('Pip required.')
-        sys.exit(1)
-
-    os.system('pip install ujson')
```
Comment (iurisilvio) [inline setup.py]: I think it removed a feature (speed). We can try to import ujson if it is available.

```
try:
    import ujson as json
except ImportError:
    import json

Comment (claudep) [inline setup.py]: When reading the numbers on https://pypi.python.org/pypi/ujson, it seems own Python json is as fast or even faster, or did I misread them?

Comment (iurisilvio) [inline setup.py]: `ujson` is faster. The numbers in their benchmarks are `number of operations`. Higher is better.

Comment (claudep) [inline setup.py]: Just found http://jyotiska.github.io/blog/posts/json_vs_simplejson_vs_ujson.html
Then let's keep the ujson import, I'll update the patch.

Comment (claudep) [inline setup.py]: Do you also want to keep the "speedups" part in setup.py?

**Comment by iurisilvio** [comment]
Yes, maintain the speedups.

Em 24 de fev de 2017 07:14, "Claude Paroz" <notifications@github.com>
escreveu:

**Comment by claudep** [comment]
Should be good to go now.

**Comment by iurisilvio** [comment]
Great! Thanks!

**Pull request merged**